src/cloudant/database.py

```diff
@@ -586,11 +590,8 @@ def missing_revisions(self, doc_id, *revisions):
         url = posixpath.join(self.database_url, '_missing_revs')
         data = {doc_id: list(revisions)}
 
-        resp = self.r_session.post(
-            url,
-            headers={'Content-Type': 'application/json'},
-            data=json.dumps(data)
-        )
+        resp = self.r_session.post(url, headers={'Content-Type': 'application/json'},
+                                   data=json.dumps(data))
```

Comment:
Sorry, I had no idea that we exceeded 1000 lines.  I think it would be fine if you upped the `max-module-lines` to 2000 in the [pylintrc](https://github.com/cloudant/python-cloudant/blob/master/pylintrc#L209).  Most of the lines in the modules are docstring anyway.